src/sentry/plugins/base/v2.py

```diff
@@ -349,15 +349,18 @@ def get_tags(self, event, **kwargs):
         """
         return []
 
-    def get_event_preprocessors(self, **kwargs):
+    def get_event_preprocessors(self, data, **kwargs):
```

Comment:
Yeah, I could tell that. I fought with myself a tiny bit on how this should be done, and this seemed to be the best given the current state of the world. I originally tossed aroudn the idea of adding another method `get_event_preprocessors_for_event` or `has_event_preprocessors`, etc. Which in theory seemed more correct, but in practice just made things a bit messier.